core/modules/utils/dom/modal.js

```diff
@@ -122,7 +122,7 @@ Modal.prototype.display = function(title,options) {
 				attributes: {
 					text: {
 						type: "string",
-						value: ( $tw.language == undefined ? "Close" : $tw.language.getString("Buttons/Close/Caption") )
+						value: $tw.language.getString("Buttons/Close/Caption")
```

Comment:
When TiddlyWiki is started in safe mode, localisable string is appeared in official TiddlyWiki plugin library.